src/components/MessageListener/MessageListener.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+
+const useListenMessage = (callback, eventType = 'trafimage-event') => {
```

Comment:
Is  this event-type 'trafimafe-event'  used somewhere ?  if not remove the default value and do nothing if there is not eventType.

Reply:
removed